layouts/page/cudf-pandas.html

```diff
@@ -159,7 +159,7 @@ <h2><i class="fa-light fa-bookmark"></i> Try Now on Colab</h2>
           {{ $colabImg := ((site.GetPage "_partners/google-colab").Resources.Get "logo.png").Resize "200x" }}
           <a href="https://nvda.ws/rapids-cudf" target="_blank"><img src="{{ $colabImg.RelPermalink }}"></a>
           <p>Take cuDF's new pandas accelerator mode for a test-drive in a free GPU-enabled notebook environment using
-            your Google account by <a href="https://nvda.ws/rapids-cudf" target="_blank">launching on CoLab <i
+            your Google account by <a href="https://nvda.ws/rapids-cudf" target="_blank">launching on Colab <i
```

Comment:
I would update the link to the blog for consistency. Would change wording to reflect it as well.